litmus-portal/frontend/src/utils/validate.ts

```diff
@@ -19,6 +19,15 @@ export const validateEmail = (value: string) => {
   return false;
 };
 
+export const validateUsername = (value: string) => {
+  const usernameValid = /^[a-zA-Z0-9\.]+$/;
```

Comment:
This regex does not allow _ @ and - in usernames, sure it excludes # but doesn't include those mentioned as well. You could please modify the regex!

Reply:
@S-ayanide I updated the RegEx. Please have a look!